ahoy.go

```diff
@@ -172,8 +174,38 @@ func getSubCommands(includes []string) []cli.Command {
 	return subCommands
 }
 
+// Given a filepath, return a string array of environment variables.
+func getEnvironmentVars(envFile string) []string {
+	var envVars []string
+
+	env, err := os.ReadFile(envFile)
+	if err != nil {
+		logger("fatal", "Invalid env file: "+envFile)
+		return nil
+	}
+
+	lines := strings.Split(string(env), "\n")
+	for _, line := range lines {
+		line = strings.TrimSpace(line)
+
+		// Ignore empty lines and comments (lines starting with '#').
+		if line == "" || strings.HasPrefix(line, "#") {
+			continue
+		}
+		envVars = append(envVars, line)
+	}
+	return envVars
+}
+
 func getCommands(config Config) []cli.Command {
 	exportCmds := []cli.Command{}
+	envVars := []string{}
+
+	// If a global environment variable file is defined, use that too.
```

Comment:
please clarify what "use that too" means exactly: will the values in the global file override the values in the local file or the other way around?

Reply:
The other way around: 'global' env files are overridden by command-level ones. 

The "use that too" is actually leftover from my original implementation, where Ahoy would read a `.env` file by default if it was present, like what docker compose currently does.